lib/sensors/src/MoistureSensor.cpp

```diff
@@ -63,18 +62,19 @@ uint16_t MoistureSensor::Read_Moisture() {
     ThisThread::sleep_for(1s);
     i2c_.read(Sensor_I2C_Address, buf, 2);  // read moisture
 
-    ret = ((uint16_t)buf[0] << 8 | buf[1]);  // concatenate bytes together
+    sensorReading = ((uint16_t)buf[0] << 8 | buf[1]);  // concatenate bytes together
```

Comment:
Shouldn't there be some conversion from the raw 16-bit data to the actual measured value (like, what are the units)?

Reply:
Unless this function is just returning the raw data?

Reply:
use static casting over c style casting